docs/drawer-actions.md

```diff
@@ -12,6 +12,7 @@ The following actions are supported:
 * [toggleDrawer](#toggleDrawer) - toggle the state, ie. switche from closed to open and vice versa
 
 ### Usage
+#TODO  rather ok, no?
```

Comment:
Looks okay. But should update the import statement since it's now in `@react-navigation/routers`